app/Http/Controllers/Bible/TextControllerV2.php

```diff
@@ -96,7 +96,7 @@ public function index($bible_url_param = null, $book_url_param = null, $chapter_
     /**
      *
      * @OA\Get(
-     *     path="/search",
+     *     path="search",
```

Comment:
It has been changed because we have two paths with the same value and swagger currently doesn't support to merge the two identical paths despite they have a different operationId. Let me know if it is ok or we need to search another solution.

You can see the same path in the file: `Http/Controllers/Bible/TextController.php line 142.`

Reply:
in TextControllerV2, the path is /text/search. Please make this change, which will remove the conflict

Reply:
@bradflood, I have updated it.